domains/shikharreyya.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "For my personal projects",
+  "repo": "https://github.com/Poseidon0070/Poseidon0070.github.io",
+  "owner": {
+    "username": "Poseidon0070",
+    "email": "criticalxsynergy@gmail.com",
+    "twitter": "Voldemort_7585"
+  },
+  "records": {
+    "CNAME": "poseidon0070.github.io"
```

Comment:
404 Site. Please send over a preview. Reopen this PR when you have this resolved.